src/Giveaway.js

```diff
@@ -289,11 +295,11 @@ class Giveaway extends EventEmitter {
         if (!reaction) return new Discord.Collection().array();
```

Comment:
I think we should do `return [];`, shouldn't we? Because is there any reason to create a empty discord collection and then return that as a array instead of just a empty JS array?

Reply:
yeah, `new Discord.Collection().array();` is useless